tests/components/DBTableMockerTest.php

```diff
@@ -15,17 +15,39 @@ class DBTableMockerTest extends TestCase {
 
         /**
          * @covers ::getTableDescription
+         * @covers ::getCurrentColumn
```

Comment:
This test now covers one more method because the assertion of that method was also added here.